src/hooks/use-estimate-gas.ts

```diff
@@ -15,20 +15,20 @@ export const useEstimateGas = (tx: EstimateContractGasParameters | null) => {
     });
   }, [chain]);
 
-  return useSWR(
-    "estimate-gas",
-    async () => {
+  React.useEffect(() => {
+    const getGas = async () => {
       try {
-        if (!tx) throw new Error("No transaction provided");
-        if (!publicClient) throw new Error("No public client available");
-        const gas = await publicClient.estimateContractGas(tx);
-        return gas;
+        if (!tx) return;
+        if (!publicClient) return;
+        const gasEstimate = await publicClient.estimateContractGas(tx);
+        setGasPrice(gasEstimate);
       } catch (error) {
-        // log error then rethrow it to be caught by SWR
-        Sentry.captureException(error);
```

Comment:
Is it still worth logging to Sentry? Do we expect the message from the error message to be enough context?

Reply:
the error that is caught here would most probably be related to a contract revert (eg, the user already claimed, not eligible or proof expired)
I think it would be better for the user to see it, rather then us. users had no feedback before about what was going wrong. we can potentially log to sentry as well but not sure if that adds too much benefit considering users sends the screen shot when they open a ticket and we would log things that are user specific errors and not an issue in general.
let me know if you disagree, we can still add it